RELEASES.md

```diff
@@ -2,54 +2,132 @@
+| yoga/100_000 nodes (5-level hierarchy) | 141.5307 ms | 1.6404 ms |
+
+(note that the table below contains multiple different units (milliseconds vs. microseconds vs. nanoseconds))
+
+While we're trying not to get too excited (there could easily be an issue with our benchmarking methodology which make this an unfair comparison), we are pleased to see that we seem to be anywhere between 100x and 1000x times faster depending on the node count!
+
+### Breaking API changes
+
+#### Node creation changes
+
+- `taffy::Node` is now unique only to the Taffy instance from which it was created.
+- Renamed `Taffy.new_node(..)` -> `Taffy.new_with_children(..)`
+- Renamed `Taffy.new_leaf()` -> `Taffy.new_leaf_with_measure()`
+- Added `taffy::node::Taffy.new_leaf()` which allows the creation of new leaf-nodes without having to supply a measure function
+
+#### Error handling/representation improvements
+
+- Renamed `taffy::Error` -> `taffy::error::TaffyError`
+- Replaced `taffy::error::InvalidChild` with a new `InvalidChild` variant of `taffy::error::TaffyError`
+- Replaced `taffy::error::InvalidNode` with a new `InvalidNode` variant of `taffy::error::TaffyError`
+- The following method new return `Err(TaffyError::ChildIndexOutOfBounds)` instead of panicking:
+  - `taffy::Taffy::remove_child_at_index`
+  - `taffy::Taffy::replace_child_at_index`
+  - `taffy::Taffy::child_at_index`
+- `Taffy::remove` now returns a `Result<usize, Error>`, to indicate if the operation was sucessful (and if it was, which ID was invalidated).
+
+#### Some uses of `Option<f32>` replaced with a new `AvailableSpace` enum
+
+A new enum `Taffy::layout::AvailableSpace` has been added.
+
+The definition looks like this:
+
+```rust
+/// The amount of space available to a node in a given axis
+pub enum AvailableSpace {
+    /// The amount of space available is the specified number of pixels
+    Definite(f32),
+    /// The amount of space available is indefinite and the node should be laid out under a min-content constraint
+    MinContent,
+    /// The amount of space available is indefinite and the node should be laid out under a max-content constraint
+    MaxContent,
+}
+```
+
+This enum is now used instead of `Option<f32>` when calling `Taffy.compute_layout` (if you previously passing `Size::NONE` to `compute_layout`, then you will need to change this to `Size::MAX_CONTENT`).
+
+And a different instance of it is passed as a new second parameter to `MeasureFunc`. `MeasureFunc`s may choose to use this parameter in their computation or ignore it as they see fit. The canonical example of when it makes sense to use it is when laying out text. If `MinContent` has been passed in the axis in which the text is flowing (i.e. the horizontal axis for left-to-right text), then you should line-break at every possible opportunity (e.g. all word boundaries), whereas if `MaxContent` has been passed then you shouldn't line break at all..
```

Comment:
Would it make sense to link to an example here?

Reply:
I would *love* more `MeasureFunc` examples in general.

Reply:
I feel like a section of the README "Integrating with other layout mechanisms" would be appropriate? Although now that I think about it, an actual example in the examples directory would also be great.

Reply:
Yeah, I think an example in the directory is the way to go here.